apps/web/lib/analytics/constants.ts

```diff
@@ -114,6 +114,7 @@ export const INTERVAL_DATA: Record<
 
 export const VALID_ANALYTICS_ENDPOINTS = [
   "count",
+  "count_new",
```

Comment:
@steven-tey are we using the separate `count_new` pipe or are we going to replace `count` with `count_new`?